### PR TITLE
Update pallet-ssvm version and README

### PR DESCRIPTION
**Pull request by dm4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,369 @@
 # Substrate SSVM Node
 
+## Build
+
+To build SSVM enabled Substrate node, we need the following repositories:
+
+```
```
Comment (CaptainVincent) [inline README.md]: add bash syntax highlight for all code snippet section
\```bash

Comment (CaptainVincent) [reply README.md]: consider use '>' notation before command. more consistency.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,369 @@
...
+docker exec -it ssvm bash
+(docker) ~/substrate-cli-tools/typescript/dist/info.js
+(docker) ~/substrate-cli-tools/typescript/dist/events.js +ssvm
+```
+
+## Deploy contract
+
+Now we could use substrate-cli-tools to deploy Ewasm contract to our node. Here we use the following ERC20 contract as an example:
+
+```
```
Comment (CaptainVincent) [inline README.md]: Attach three files and point source code sol(ERC20.sol) link url
1. ERC20.sol
2. ERC20.wasm
3. ERC20.hex

File: README.md [hunk 3]
```diff
@@ -1,5 +1,369 @@
...
+
+    constructor() public {
+        _name = "ERC20TokenDemo";
+        _symbol = "ETD";
+        _mint(msg.sender, 1000);
+    }
+}
+```
+
+First, we need to compile the Solidity contract to wasm bytecode using [SOLL](https://github.com/second-state/soll):
```
Comment (CaptainVincent) [inline README.md]: Describe three files relationship.
We also can keep and use those command help us describe their relationship,
but not guide user do that by himself.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,369 @@
...
+```
+xxd -p erc20.wasm | tr -d $'\n' > erc20.hex
+```
+
+Now we could deploy the wasm bytecode to our node:
+
+```
+# Start a new shell in the same docker container
+docker exec -it ssvm bash
+(docker) ~/substrate-cli-tools/typescript/dist/ssvm.js create -p 99 -g 5000000 -c 0x$(cat erc20.hex)
```
Comment (CaptainVincent) [inline README.md]: maybe use other identifier denote this is different container.

Comment (dm4) [reply README.md]: Is it okay to use `docker-1`, `docker-2` to identify different shells?

Comment (CaptainVincent) [inline README.md]: Add tip make sure user establish/run a ssvm-node before interactive it.

File: README.md (hunk 2 above)
Comment (dm4) [reply README.md]: Is it okay to put these files into `erc20/` directory at project root?

File: README.md (hunk 4 above)
Comment (dm4) [reply README.md]: Okay, I'll guide user execute `info.js` to make sure rpc server is running

**Comment by CaptainVincent** [comment]
LGTM.

**Pull request merged**